flutter_local_notifications/lib/src/platform_specifics/android/status_bar_notification.dart

```diff
@@ -0,0 +1,22 @@
+class ActiveNotification {
+  const ActiveNotification(
+    this.id,
+    this.channelId,
+    this.title,
+    this.body,
+  );
+
+  final int id;
```

Comment:
Missing API docs for the property. Can refer to the existing public API docs to copy and paste them if it makes sense to do so. `channelId` will need special mention that it is only returned on Android 8.0+

Reply:
Ok I added documentation comments to all the properties. Let me know if it's ok like that